internal/datastore/postgres/postgres.go

```diff
@@ -188,7 +189,13 @@ func NewPostgresDatastore(
 		config.gcWindow.Seconds(),
 	)
 
+	maxRevisionStaleness := time.Duration(float64(config.revisionQuantization.Nanoseconds())*
+		config.maxRevisionStalenessPercent) * time.Nanosecond
```

Comment:
isn't duration already in nanoseconds?

Reply:
Yes, this just makes it explicit. They must want you to do this otherwise `time.Nanosecond` wouldn't exist!